esphome/components/seeed_mr60bha2/sensor.py

```diff
@@ -40,6 +42,9 @@
             accuracy_decimals=2,
             icon=ICON_SIGNAL,
         ),
+        cv.Optional(CONF_TARGET_NUM): sensor.sensor_schema(
```

Comment:
I feel this should be the other way around
```suggestion
        cv.Optional(CONF_NUM_TARGETS): sensor.sensor_schema(
```